test/unit/ConfigProviderTest.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Boesing\Psr\Http\Message\Multipart;
 
+use Boesing\Psr\Http\Message\Multipart\ConfigProvider;
```

Comment:
shouldn't be necessary as the namespace is already `Boesing\Psr\Http\Message\Multipart` but I guess that has been added by codesniffer? 🤷🏼‍♂️